userguide/dailyuse/libertine.rst

```diff
@@ -105,78 +115,95 @@ Tips
 
   libertine-container-manager configure -i CONTAINER-IDENTIFIER -b add -p /media/phablet/ID-OF-SD/directory/you/want
   
-This will not allow the container access to any of the directories earlier in the path for anything other than accessing your mounted directory.
+This will not give the container access to any of the directories earlier in the path for anything other than accessing your mounted directory.
     
-In order to use the SD-card as extra space for your container, make sure first to format it using ext4 or similar.
-There is a mis-feature in udisk2 that mounts SD-cards (showexec) that ensures only files ending in .bat, .exe or .com can be executed from the drive if it is (v)fat formatted. This has been changed in other distributions allowing any file to have execute priviliges, but not ubuntu. The reccomended workaround is to add a udev rule to control how to mount a card with a given id, but since the udev rules are on the read only port on touch, this is not possible.
+In order to use the SD card as extra space for your container, make sure to first format it using ext4 or similar.
+There is a mis-feature in UDisks2 that mounts SD-cards (showexec), ensuring only files ending in .bat, .exe or .com can be executed from the drive if it is (V)FAT formatted.
+This has been changed in other distributions, allowing any file to have execute priviliges, but not in Ubuntu.
+The recommended workaround is to add a udev rule to control how to mount a card with a given ID, but since the udev rules are on the read-only port on Ubuntu Touch, this is not possible.
 
 Shortcuts
 ^^^^^^^^^
 
-If you want, you can add aliases for command line tools. Add lines like the following ones to your ``~/.bash_aliases``::
+If you want, you can add aliases for command-line tools.
+Add lines like the following ones to your ``~/.bash_aliases``::
 
     alias git='libertine-launch -i CONTAINER-IDENTIFIER git'
     alias screenfetch='libertine-launch -i CONTAINER-IDENTIFIER screenfetch'
 
 Background
 ----------
 
-A display server coordinates input and output of an operating system. Most Linux distributions today use the X server. Ubuntu Touch does not use X, but  a new display server called Mir. This means that standard X applications are not directly compatible with Ubuntu Touch. A compatibility layer called XMir resolves this. Libertine relies on XMir to display desktop applications.
-
-Another challenge is that Ubuntu Touch system updates are released as OTA images. A consequence of this is that the root filesystem is read only. Libertine provides a container with a read-write filesystem to allow the installation of regular Linux desktop applications.
+A display server coordinates input and output of an operating system.
+Most Linux|GNU distributions today use the X server.
+Ubuntu Touch does not use X, but a new display server called Mir.
+This means standard X apps are not directly compatible with Ubuntu Touch.
```

Comment:
applications

Reply:
programs programs programs

Reply:
I don't understand. Are you suggesting to use the word "programs" instead of "applications"/"apps". Fine with me. 

Either way, right now I count the words
* "application" 7 times, and 
* "apps" 7 times 
when referring to Libertine applications

Please use one of "program" or "application" in all 14 cases.

Reply:
Didn't I get shut down on calling it "programs"?
My rationale is that there is no distinction between (mobile) app, and (desktop) program anymore.
Light programs on desktop are widgets, and that seems to have made its way onto mobile platforms too, so "apps" only has this possible mismatch.

I seemingly only changed more of the "application" ones to "app".

Reply:
> Didn't I get shut down on calling it "programs"? 

oh, did I. Can't remember. It's been a long PR. Sorry, for the back and forth then

> Pretty sure it isn't "programs" anywhere else.
> 
> My rationale is that there is no distinction between (mobile) app, and (desktop) program anymore.
> Light programs on desktop are widgets, and that seems to have made its way onto mobile platforms too, so "apps" only has this possible mismatch.
> 

Well, my rationale is that there is still a pretty significant distinction between mobile apps and desktop programs *when dealing with Libertine*. 

> I seemingly only changed more of the "application" ones to "app".
> 
> Small problem compared to the greater wealth of changes.
> 
> Edit: Seems like it is more often "applications" for desktop, and "apps" for mobile?

Yep. My idea when writing it and in this review was to use/keep these two words in these two contexts.